src/views/Clubs.vue

```diff
@@ -6,12 +6,12 @@
         <v-row>
           <v-col
             v-for="item in clubInfo[index - 1].clubs"
-            :key="item.engName"
+            :key="item.id"
             align="center"
           >
-            <v-card width="370" align="left" :to="`/clubpage/${item.engName}`">
+            <v-card width="370" align="left" :to="`/clubpage/${item.id}`">
```

Comment:
Persist to their English names, since we want more human-readable URLs (e.g. http://theenspire.com/clubpage/WFLA%20Volleyball%20Club)

Reply:
Done.